server/app.ts

```diff
@@ -0,0 +1,19 @@
+import express, { Express, json, urlencoded } from 'express';
+
+import cookieParser from 'cookie-parser';
+import compression from 'compression';
+
+import { router } from './routes/index';
+
+const app: Express = express();
+
+app.use([
+  json(),
+  urlencoded({ extended: false }),
+  compression(),
+  cookieParser(),
+]);
```

Comment:
Use cors middleware to allow other origins to reach to our server. http://localhost:5173 is seend as other origin.